scripts/templates/component.ts

```diff
@@ -4,15 +4,15 @@ import ComponentFileBuilderResponse from "./ComponentFileBuilderResponse";
 export default (componentName: string, componentType: string): ComponentFileBuilderResponse => ({
     content: `/* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import { ReactElement } from "react";
+import { FC } from "react";
 
-export interface ${componentName}Props {
+export type ${componentName}Props = {
     foo: string;
-}
+};
 
-export default function ${componentName}({ foo }: ${componentName}Props): ReactElement<${componentName}Props> {
+export const ${componentName}: FC<${componentName}Props> = ({ foo }) => {
```

Comment:
```suggestion
export const ${componentName}: FC<${componentName}Props> = ({ foo }: ${componentName}Props) => {
```

Reply:
no need since the `FC` type ensures that props have the correct typing